src/CabanaPD_Solver.hpp

```diff
@@ -158,7 +166,7 @@ class SolverElastic
         if ( print )
         {
             log( std::cout, "Local particles: ", particles->n_local,
-                 ", Maximum local neighbors: ", max_neighbors );
+                 ", Maximum neighbors: ", max_neighbors );
```

Comment:
@streeve : Why do we have both here and in Line 185 the output of "Maximum neighbors: ", max_neighbors "?

Reply:
This prints to screen, the other prints to file